spiketoolkit/comparison/gathersortingcomparison.py

```diff
@@ -6,7 +6,7 @@
 from spiketoolkit.sorters import run_sorters, collect_results
 from .sortingcomparison import compare_two_sorters, _perf_keys
 
-def gather_sorting_comparison(working_folder, ground_truths, use_multi_index=True):
+def gather_sorting_comparison(working_folder, ground_truths, use_multi_index=True, threshold=95):
```

Comment:
Can we call this accuracy_threshold or acc_threshold? Otherwise, it is a bit confusing what it is thresholding.

Reply:
Also, does 95 seem a bit high? Maybe 90 or something? Not sure we want a threshold that high as default? I don't really know though